packages/circuit-ui/components/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -254,27 +295,31 @@ export const PhoneNumberInput = forwardRef<
           <Select
             hideLabel
             autoComplete="tel-country-code"
-            invalid={invalid}
+            invalid={invalid || countryCode.invalid}
             disabled={disabled}
             className={classes['country-code']}
+            aria-readonly={readOnly || countryCode.readonly}
             {...countryCode}
```

Comment:
The `invalid` and `aria-readonly` props will be overwritten by the spread `{...countryCode}` props below. Move them below to avoid this.

Reply:
Code updated